docs: Document PDF author and keywords (#132)
slidevjs/docs: custom/index.md

title: 'Slidev'
# titleTemplate for the webpage, `%s` will be replaced by the page's title
titleTemplate: '%s - Slidev'

> # information for your slides, can be a markdown string.  Also used as the PDF Title
> info: false
> # Populates the PDF Author field
> author: Your Name Here
> # Populates the PDF Keywords field. Comma-delimited
> keywords: keyword1,keyword2

Reviewer: ```suggestion
# information for your slides, can be a markdown string.
info: false
# author field for exported PDF
author: Your Name Here
# keywords field for exported PDF, comma-delimited.
keywords: keyword1,keyword2
```
